src/swagger-specs/destinations.yaml

```diff
@@ -34,81 +34,313 @@ servers:
     description: Adobe IO gateway endpoint
 
```

Comment:
The top-level `description` on line 3 should have something that explains that this reference focused on the destination-based use cases for Flow Service API, and provide a link to the sources side in case that's what the reader is actually looking for.